src/main/webapp/blobstoreFunctions.js

```diff
@@ -0,0 +1,38 @@
+/** Loads action URL and attaches it to the upload form (the user's browser uploads the file to blobstore with the generated url) */
+function fetchBlobstoreUrl() {
+  fetch('/blobstore-upload-url').then((response) => response.text()).then((imageUploadUrl) => {
+    document.getElementById('plant-pic-form').action = imageUploadUrl;
+  });
+}
+ 
+/**
+  * Creates and adds a card element for the photo 
+  * pic_url is the URL of image to be shown in the post
+*/
+function createListElement(pic_url) {
+  let card = document.createElement("div");
+  let body = document.createElement("div");
```

Comment:
body is ambiguous because HTML has a body tag.

Reply:
Got it, I changed it to card_body.